src/azure-cli/azure/cli/command_modules/profile/__init__.py

```diff
@@ -40,43 +39,67 @@ def load_command_table(self, args):
 
         return self.command_table
 
-    # pylint: disable=line-too-long
     def load_arguments(self, command):
         from azure.cli.core.api import get_subscription_id_list
 
         with self.argument_context('login') as c:
-            c.argument('password', options_list=['--password', '-p'], help="Credentials like user password, or for a service principal, provide client secret or a pem file with key and public certificate. Will prompt if not given.")
-            c.argument('service_principal', action='store_true', help='The credential representing a service principal.')
-            c.argument('username', options_list=['--username', '-u'], help='user name, service principal, or managed service identity ID')
-            c.argument('tenant', options_list=['--tenant', '-t'], help='The AAD tenant, must provide when using service principals.', validator=validate_tenant)
-            c.argument('allow_no_subscriptions', action='store_true', help="Support access tenants without subscriptions. It's uncommon but useful to run tenant level commands, such as 'az ad'")
+            c.argument('username', options_list=['--username', '-u'],
+                       help='User name, service principal client ID, or managed identity ID.')
+            c.argument('password', options_list=['--password', '-p'],
+                       help='Credentials like password for user, or for a service principal, provide client '
+                            'secret or a PEM file with key and public certificate. Will prompt if not given.')
+            c.argument('tenant', options_list=['--tenant', '-t'], validator=validate_tenant,
+                       help='The Microsoft Entra tenant, must provide when using a service principal.')
```

Comment:
Should we change it to this?

```suggestion
                       help='The Microsoft Entra tenant, must be provided when using a service principal.')
```